lib/rbi/model.rb

```diff
@@ -42,6 +42,16 @@ def replace(node)
       self.parent_tree = nil
     end
 
+    sig { params(nodes: T::Enumerable[Node]).void }
+    def replace_with_multiple(nodes)
+      tree = parent_tree
+      raise unless tree
+
+      # Does this work?
```

Comment:
Did you find a way to answer this question?

Reply:
Yup, works :)